docs/usage/examples/opentelemetry.md

```diff
@@ -0,0 +1,278 @@
+latency_bucket{http_host="api.github.com:443",http_method="POST",http_status_code="200",operation="HTTPS POST",service_name="renovate",span_kind="SPAN_KIND_CLIENT",status_code="STATUS_CODE_UNSET",le="+Inf"} 9
+latency_sum{http_host="api.github.com:443",http_method="POST",http_status_code="200",operation="HTTPS POST",service_name="renovate",span_kind="SPAN_KIND_CLIENT",status_code="STATUS_CODE_UNSET"} 2306.1385999999998
+latency_count{http_host="api.github.com:443",http_method="POST",http_status_code="200",operation="HTTPS POST",service_name="renovate",span_kind="SPAN_KIND_CLIENT",status_code="STATUS_CODE_UNSET"} 9
+latency_bucket{http_host="gitlab.com:443",http_method="GET",http_status_code="401",operation="HTTPS GET",service_name="renovate",span_kind="SPAN_KIND_CLIENT",status_code="STATUS_CODE_ERROR",le="0.1"} 0
+latency_bucket{http_host="gitlab.com:443",http_method="GET",http_status_code="401",operation="HTTPS GET",service_name="renovate",span_kind="SPAN_KIND_CLIENT",status_code="STATUS_CODE_ERROR",le="1"} 0
+latency_bucket{http_host="gitlab.com:443",http_method="GET",http_status_code="401",operation="HTTPS GET",service_name="renovate",span_kind="SPAN_KIND_CLIENT",status_code="STATUS_CODE_ERROR",le="2"} 0
+latency_bucket{http_host="gitlab.com:443",http_method="GET",http_status_code="401",operation="HTTPS GET",service_name="renovate",span_kind="SPAN_KIND_CLIENT",status_code="STATUS_CODE_ERROR",le="6"} 0
+latency_bucket{http_host="gitlab.com:443",http_method="GET",http_status_code="401",operation="HTTPS GET",service_name="renovate",span_kind="SPAN_KIND_CLIENT",status_code="STATUS_CODE_ERROR",le="10"} 0
+latency_bucket{http_host="gitlab.com:443",http_method="GET",http_status_code="401",operation="HTTPS GET",service_name="renovate",span_kind="SPAN_KIND_CLIENT",status_code="STATUS_CODE_ERROR",le="100"} 0
+latency_bucket{http_host="gitlab.com:443",http_method="GET",http_status_code="401",operation="HTTPS GET",service_name="renovate",span_kind="SPAN_KIND_CLIENT",status_code="STATUS_CODE_ERROR",le="250"} 1
+latency_bucket{http_host="gitlab.com:443",http_method="GET",http_status_code="401",operation="HTTPS GET",service_name="renovate",span_kind="SPAN_KIND_CLIENT",status_code="STATUS_CODE_ERROR",le="9.223372036854775e+12"} 1
+latency_bucket{http_host="gitlab.com:443",http_method="GET",http_status_code="401",operation="HTTPS GET",service_name="renovate",span_kind="SPAN_KIND_CLIENT",status_code="STATUS_CODE_ERROR",le="+Inf"} 1
+latency_sum{http_host="gitlab.com:443",http_method="GET",http_status_code="401",operation="HTTPS GET",service_name="renovate",span_kind="SPAN_KIND_CLIENT",status_code="STATUS_CODE_ERROR"} 232.9222
+latency_count{http_host="gitlab.com:443",http_method="GET",http_status_code="401",operation="HTTPS GET",service_name="renovate",span_kind="SPAN_KIND_CLIENT",status_code="STATUS_CODE_ERROR"} 1
+```
```

Comment:
The above is pretty lengthy (at least in markdown source). Is there any alternative to having so much text?

Reply:
I was thinking the same thing. Why are we providing this example? What does the user do with it, do they even need to know it?

Reply:
When I was testing it out, I ended up creating the a `spanmetricsprocessor` as well and found the information from it quite handy. So I do think the section generally makes sense, but yeah the example can maybe shorten to one or two example metric lines? Especially since (which I find really nice) explain them in more detail later on anyways.

Reply:
I have added them to show full spectrum of label combinations that are possible, but I agree that it is lengthy. 
The example block is now shortened and includes instead some commentary